### PR TITLE
PR: Remove Windows/Py2.7 restriction

### LINKED ISSUE #131: Remove compatibility check for Windows and Python 2
Now that pywinpty is available for that combination, there's no need of that check.

### PR DESCRIPTION
**Pull request by andfoy**
Fixes #131

### CODE REVIEW

**Comment by andfoy** [comment]
@ccordoba12 We have several segmentation fault errors both on Linux and Windows

**Comment by ccordoba12** [comment]
There are two options:

1. Skip the faulty tests and investigate why they are failing later.
2. Pin Qt/PyQt to 5.6.

Option 2. doesn't seem too realistic though, if we want people to keep using spyder-terminal with the latest conda packages.

**Comment by andfoy** [comment]
@ccordoba12 Under the current setup, we cannot even start a Qt application. Except for the CircleCI instance that does download and install the PyQt wheels from PyPi

**Comment by ccordoba12** [comment]
> Under the current setup, we cannot even start a Qt application

What does this mean?

**Comment by andfoy** [comment]
@ccordoba12 It looks like the Qt/PyQt installation installed via conda is corrupted

**Comment by ccordoba12** [comment]
@andfoy, please remove your last two commits, rebase on top of master and add Python 2.7 to the Appveyor matrix.

File: appveyor.yml.orig [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# https://ci.appveyor.com/project/spyder-ide/spyder-terminal
```
Comment (ccordoba12) [inline appveyor.yml.orig]: Please remove this file and add orig files to gitignore.

**Comment by andfoy** [comment]
@ccordoba12 We're still having problems with Qt on Windows and Python 2.7

**Comment by ccordoba12** [comment]
@andfoy, you're right, so I commented the Python entry in Appveyor.

I also cleaned and rebased your PR, and I'll merge it as soon as it passes (please don't add more commits here).

**Pull request merged**